File: Python/7kyu/Regex validate PIN code.py
# https://www.codewars.com/kata/55f8a9c06c018a0d6e000132/train/python
import re


def validate_pin(pin):
    # return true or false
    p = re.findall(r"(^\d{6})|(^\d{4})", pin)
    if p:
        return max(p[0]) == pin
    return False

print(validate_pin('1234'))

File: JavaScript/7kyu/Regex validate PIN code.js
// https://www.codewars.com/kata/55f8a9c06c018a0d6e000132/train/javascript
function validatePIN (pin) {
    let p = pin.search(/(^\d{6}$)|(^\d{4}$)/gi)
    console.log(p)
    if (p>=0) {
        return true
    }
    return false
}

const pin = ("12345")
console.log(validatePIN (pin))